application/models/Question.php

```diff
@@ -547,6 +547,7 @@ public function getQuestionList($surveyid)
 
     /**
      * @return string
+     * NOTE: Not used anymore. Based on a deprecated method. Should be deprecated.
```

Comment:
Ehm not used = should be removed then? Why keep code that's not used?

Reply:
Well, it is a public method. I would wait a bit before removing it.
Someone could be using it on a custom development.

Reply:
This is not **this** commit that deprecate the function, tight ?

Reply:
No, if recall correctly, we just added the comment as we found that out

Reply:
Then : i think it's a good idea to add comments, but not to remove or deprecate on a unrelated commit :)

Reply:
Yes, as we did

Reply:
> Yes, as we did

Exactly :)